charts/llama-stack/templates/hpa.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
I doubt llama-stack's server supports running multiple replicas, especially with the local SQLite DBs, unless you have a PVC with RWX.

How about we remove this for now?

Reply:
We can... but tbh I don't see how we could call this production-ready before we add the capabilities to externalize the data source. At which point I think we are just going to want to add this back in. 

Would it make more sense to just remove the documentation on it? If not I am not that married to the HPA and I can make a comment on the replicas that multi replica will have un-expected results if you have more than one.

Reply:
I’d prefer not to introduce content based on the promise of future delivery, as we can never be certain when it will actually happen. Although a discussion started here: https://github.com/meta-llama/llama-stack/discussions/1681.

I think removing the file and adding a comment to the replica field is sufficient for now.
Thanks!